### PR TITLE
Making Database autogenerated Id mandatory in Type definition

### LINKED ISSUE #162: Look into making `{model}.id` not optional in type definitions
Thus far, model definitions (`user.model.ts`, `canvas.model.ts`) have used `id?: bigint` for their types, as seen here:
https://github.com/tl-its-umich-edu/canvas-course-manager-next/blob/9922493e8df0b7fefae5949ce77b0cfce88311f1/ccm_web/server/src/user/user.model.ts#L7-L13

This seems wrong to me on reflection. `id` should always be defined.

### PR DESCRIPTION
**Pull request by pushyamig**
Fixes #162

### CODE REVIEW

File: ccm_web/server/src/user/user.model.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { CanvasToken } from '../canvas/canvas.model'
 // https://www.npmjs.com/package/sequelize-typescript#more-strict
 
 interface UserAttributes {
-  id?: bigint
+  id: bigint
```
[the remark this answers is not in the document]
Comment (pushyamig) [reply ccm_web/server/src/user/user.model.ts:8]: Done

**Comment by pushyamig** [comment]
@ssciolla I resolved all the issue you raised.

**Review by ssciolla — APPROVED** [review]
Okay, this looks good to me! I'm still not 100 percent sure what the correct way to do all this is, but because we use `bigint`, I think maybe we need to declare the `id` column ourselves like this. I think we can move forward this and continue to learn. Thanks!

**Comment by zqian** [comment]
Verified the `user` and `canvas_token` table has `id` field as type `bigint`, and also defined as Primary Key

**Pull request merged**